lib/statsig/evaluator.ex

```diff
@@ -434,24 +438,31 @@ defmodule Statsig.Evaluator do
     hash
   end
 
-  defp get_user_id(user, "userID"), do: try_get_with_lower(user, "userID") |> to_string()
+  defp get_user_id(%User{userID: id}, "userID"), do: to_string(id)
 
-  defp get_user_id(user, prop),
-    do: try_get_with_lower(Map.get(user, "customIDs", %{}), prop) |> to_string()
+  defp get_user_id(%User{customIDs: custom_ids}, prop),
```

Comment:
remember, do not pattern match in parameters unless it affects which function head is being called. This can be written as:
```suggestion
  defp get_user_id(%User{} = user, prop),
    do: try_get_with_lower(user.custom_ids, prop)
```